esp32/modules/appglue.py

```diff
@@ -21,7 +21,8 @@ def start_ota():
     esp.rtcmem_write(1,254)
     deepsleep.reboot()
 
-def start_bpp():
+def start_bpp(duration):
```

Comment:
Was left like this to give sprite the chance to make his pr. Which, once merged, makes this whole function completely useless.